### PR TITLE
Separate validate -print_usage into separate tool

### PR DESCRIPTION
**Pull request by rakillen**
Internal JIRA 388

README updates will be a separate PR.

### CODE REVIEW

File: installer/src/main/bin/validateModel.cmd [hunk 1]
```diff
@@ -65,7 +74,6 @@ if "%SHOW_USAGE%" == "false" (
 ECHO.
 ECHO Usage: %SCRIPT_NAME% [-help]
 ECHO              -oracle_home ^<oracle_home^>
-ECHO              [-print_usage ^<context^> [-attributes_only^|-folders_only^|-recursive] ]
 ECHO              [-model_file ^<model_file^>]
```
Comment (CarolynRountree) [inline installer/src/main/bin/validateModel.cmd:77]: model file is now required for validate (print usage you did not give model file)

File: installer/src/main/bin/validateModel.sh [hunk 2]
```diff
@@ -31,7 +31,6 @@ usage() {
   echo ""
   echo "Usage: $1 [-help]"
   echo "          -oracle_home <oracle_home>"
-  echo "          [-print_usage <context> [-attributes_only|-folders_only|-recursive] ]"
   echo "          [-model_file <model_file>]"
```
Comment (CarolynRountree) [inline installer/src/main/bin/validateModel.sh:34]: same as above

Comment (rakillen) [reply installer/src/main/bin/validateModel.sh:34]: Model can be in the archive file

File: installer/src/main/bin/validateModel.cmd (hunk 1 above)
Comment (rakillen) [reply installer/src/main/bin/validateModel.cmd:77]: Model can be in the archive file

**Pull request merged**